shared/chat/conversation/messages/set-explode-popup/index.desktop.js

```diff
@@ -104,6 +104,7 @@ export default (props: Props) => {
   return (
     <FloatingMenu
       attachTo={props.attachTo}
+      position="top center"
```

Comment:
Suggestion from @buoyad to explicitly set the floating menu position to `top center` such that if the position doesn't calculate, the floating box won't render below the input area.

Defaults to `bottom center`: https://github.com/keybase/client/blob/master/shared/common-adapters/floating-box.desktop.js#L50

This doesn't change the position on desktop or mobile from my testing.